src/modules/RootPart.rb

```diff
@@ -1805,8 +1805,8 @@ def MountPartitions(root_device_current)
         # in the target map of the storage module
         AddToTargetMap()
         if Yast2::FsSnapshot.configured?
-          snapshot = Yast2::FsSnapshot.create_pre("before upgrade")
-          Yast2::FsSnapshotStore.save("upgrade", snapshot.number)
+          snapshot = Yast2::FsSnapshot.create_pre("before update", cleanup: :number, important: true)
+          Yast2::FsSnapshotStore.save("update", snapshot.number)
```

Comment:
from IRC (just for the record):

[11:31:57] <HuHa> jreidinger: as for upgrade vs. update: I remember this discussion back a long time ago, and it seems to keep coming back
[11:32:09] <HuHa> jreidinger: whatever terminology we choose, it should be consistent, which here it is not
[11:32:20] * mvidner 0x0běd
[11:32:45] <HuHa> if we call the module and the process and everything around it "update", it's utterly inconsistent to name the snapshot done before and after it "upgrade"
[11:33:00] <HuHa> if we choose another terminology, fine by me
[11:33:07] <HuHa> but it should really be consistent